File: problems/r/romans/romans.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    x = float(input())
    roman = x * 1000 * 5280/4854
    print(round(roman))


if __name__ == '__main__':
    main()

File: problems/r/romans/romans.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>
#include <cmath>

using namespace std;

int main() {
    float x;
    int roman;

    cin >> x;

    roman = round(x * 1000 * 5280/4854);

    cout << roman << endl;
}